lib/rules/strict.js

```diff
@@ -17,7 +17,9 @@ var messages = {
     global: "Use the global form of \"use strict\".",
     multiple: "Multiple \"use strict\" directives.",
     never: "Strict mode is not permitted.",
-    unnecessary: "Unnecessary \"use strict\" directive."
+    unnecessary: "Unnecessary \"use strict\" directive.",
+    unnecessaryInModules: "Unnecessary \"use strict\" directive, because inside of modules.",
```

Comment:
@michaelficarra 

> Why do these messages need to be different from the regular "unnecessary" message? 

Because feedbacks of #2785, #2802, and #2957. Developers are not always specialist of the language spec, so those might be confused by just `unnecessary`.

@nzakas Thank you for correction!